satellite/peer.go

```diff
@@ -181,6 +181,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 	var err error
 
 	{ // setup listener and server
+		zap.S().Debug("Starting listener and server")
```

Comment:
Sorry, I've just noticed the there is a logger `log` passed to the constructor. So the corrector logging should be done via `log.Debug(...)`.